appveyor/install.ps1

```diff
@@ -1,24 +1,33 @@
 # check that all the required ODBC drivers are available, and install any that are missing
 
 Function DownloadFileFromUrl ($url, $file_path) {
+    $curl_params = "-f -sS -L -o `"$file_path`" `"$url`""
+    If (${env:APVYR_VERBOSE} -eq "true") {
```

Comment:
nitpick:  all the other code uses the lowercase form of built-in functions.

```suggestion
    if (${env:APVYR_VERBOSE} -eq "true") {
```

Reply:
+1 - may as well be consistent

Reply:
Will do.